gravity.sh

```diff
@@ -216,6 +216,22 @@ gravity_DownloadBlocklistFromUrl() {
 
   str="Status:"
   echo -ne "  ${INFO} ${str} Pending..."
+  # Determine if the domain is blocked by Pi-hole
+  if [ `dig $domain +short | grep 0.0.0.0 -c` -ge 1 ]; then
```

Comment:
>` grep 0.0.0.0`

User may not be using the NULL block mode. What if the result is `nxdomain` or the IP of the Pi-hole?

Reply:
Now covering all blockmodes

Reply:
Now covering all block modes